linode/manager
# refactor: [M3-6274] - MUI v5 Migration - `SRC > Features > Longview` Pt2: Longview Landing and Root folder

## Description 📝
**Note: the branch r-m3-6274-p2 branches off from r-m3-6274, so there are a lot of changed files here. Adding the wait to merge labels as the [part 1 pr](https://github.com/linode/manager/pull/9600) being merged first should reduce the changed file diff here. You can see the diff between this branch and the part1 branch [here](https://github.com/coliu-akamai/manager/pull/2)
 - MUI v5 migration for the rest of the Longview folder

## Major Changes 🔄
 - migrate to MUIv5 -- use styled components
 - remove React.FC
 - cleanup imports and exports

## How to test 🧪
1. Navigate to the Longview Landing 
2. confirm that everything still looks the same*
3. Confirm that things still work as expected
* one slight difference:
this branch -- gauges now centered in the row (which looks a bit better)
![image](https://github.com/linode/manager/assets/139280159/5dda221b-5f82-4e9a-bd5d-29b2660cacf6)
prod -- the gauges weren't centered on prod for some reason
![image](https://github.com/linode/manager/assets/139280159/a8598327-2e13-4938-a1a8-f82066f468c9)
